docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -63,53 +62,101 @@ spec:
           context: ./
           dockerfileUrl: Dockerfile
           revision: "{{.version}}"
-          url: https://github.com/ifireball/konflux-multibranch-sample.git
+          url: https://github.com/konflux-ci/multi-version-konflux-sample.git
+
+  - apiVersion: appstudio.redhat.com/v1alpha1
+    kind: ImageRepository
+    metadata:
+      name: imagerepository-for-multi-version-konflux-sample-{{.versionName}}
+      labels:
+        appstudio.redhat.com/application: multi-version-konflux-sample-{{.versionName}}
+        appstudio.redhat.com/component: multi-version-konflux-sample-{{.versionName}}
+    spec:
+      image:
+        name: konflux-samples-tenant/multi-version-konflux-sample-{{.versionName}}/multi-version-konflux-sample-{{.versionName}}
+        visibility: public
+      notifications:
+        - config:
+            url: 'https://bombino.preprod.api.redhat.com/v1/sbom/quay/push'
+          event: repo_push
+          method: webhook
+          title: SBOM-event-to-Bombino
+
 ----
 
 The *resources* section for the template may be created by looking at the YAML for existing resources and copying it while removing generated and unnecessary data and adding variable references where needed.
 
+The following kinds of resources are currentyl supported. Please do not attempt to create other kinds as that would prevent the template from being applied successfully.
+
+[cols="2,1,3"]
+|===
+|apiVersion |kind |Comments
+
+|appstudio.redhat.com/v1alpha1
+|Application
+|
+
+|appstudio.redhat.com/v1alpha1
+|Component
+|
+
+|appstudio.redhat.com/v1alpha1
+|ImageRepository
+|Every component should have an accompanying `ImageRepository` resource
+
+|appstudio.redhat.com/v1beta2
+|IntegrationTestScenario
+|Note the `v1beta2` API version. Make sure to use the right version when querying the cluster, otherwise important data may be missing from the resutls you get.
+
+|appstudio.redhat.com/v1alpha1
+|ReleasePlan
+|
+|===
+
+*Note:* When in doubt - the source of truth about what the YAML for your resources should look like is the up to date YAML you can find on the Konflux clusters. We try to keep the exaples here up to date, but with this being a static document there may be a gap between what you see here and what you should configure in tour system. It is recommended that you follow the process of creating the template by copying the YAML of the resrouces you have and cleaning it as describe blow, and not by copying the sample you see here.
+
 Here are specific examples for how to clean up and use the YAML for certain resource kinds:
 
-* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid` and `ownerReferences`, metadata fields should be removed as well as the `status` section.
+* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid`, `ownerReferences` and `finalizers` metadata fields should be removed as well as the `status` section.
```

Comment:
```suggestion
* For any kind of resource used by the Project Controller, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid`, `ownerReferences`, and `finalizers` metadata fields should be removed as well as the `status` section.
```

Reply:
No, this is not about the project controller using the resources, this is about how to specify them in the template.